src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrinterSettings.Windows.cs

```diff
@@ -650,7 +650,7 @@ public Graphics CreateMeasurementGraphics(PageSettings pageSettings, bool honorO
         private static Interop.Comdlg32.PRINTDLGX86 CreatePRINTDLGX86()
         {
             Interop.Comdlg32.PRINTDLGX86 data = new Interop.Comdlg32.PRINTDLGX86();
-            data.lStructSize = Marshal.SizeOf(typeof(Interop.Comdlg32.PRINTDLGX86));
+            data.lStructSize = Marshal.SizeOf<Interop.Comdlg32.PRINTDLGX86>();
```

Comment:
`PRINTDLG` looks like a good candidate for a blittable structure. (The string fields that prevent it from being blittable are not actually for anything; they just need to be there to maintain the layout.)

Reply:
It seems that `PRINTDLGX86` and `PRINTDLG` can be folded into single structure. It seems that if I add `, Pack = 1` then I can get rid of `PRINTDLGX86` and do not have switch on `IntPtr.Size` in couple places.

Reply:
I do not think that it would work. There would be a problem with padding after the short fields.

Note the Windows SDK headers have this ifdef around definitions of these structs:
```
#if !defined(_WIN64)
#include <pshpack1.h>         /* Assume byte packing throughout */
#endif
```
The dual definition of these structs is there to replicate what this ifdef does.